### PR TITLE
Update common-pitfalls.md

### PR DESCRIPTION
**Pull request by jacksorjacksor**
Have done this for Umbraco 15 docs only. Can go back through earlier docs and copy/paste in the updates if the points are relevant there too.

## Description

common-pitfalls.md

## Type of suggestion

* [X] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
Umbraco 15 (and earlier)

## Deadline (if relevant)
N/A

### CODE REVIEW

**Comment by sofietoft** [comment]
Thanks for the PR @jacksorjacksor !
I'll make sure we get it reviewed as soon as possible!

File: 15/umbraco-cms/reference/common-pitfalls.md [hunk 1]
```diff
@@ -206,18 +206,16 @@ Some examples of what not to do, and what to do instead:
 
 ## Processing during startup
 
-Umbraco allows you to run some initialization code during startup by using `UmbracoApplicationStartingNotification`. Depending on what code is run, it can have an impact on the application startup. This is especially true for Package developer as your code could end up impacting many websites.
+Umbraco allows you to run some initialization code during startup by using `UmbracoApplicationStartingNotification`. This codee can have a negative impact on the application startup process. This is especially true for Package developers as your code could end up impacting many websites.
```
Comment (sofietoft) [inline 15/umbraco-cms/reference/common-pitfalls.md:209]:
```suggestion
Umbraco allows you to run some initialization code during startup by using `UmbracoApplicationStartingNotification`. This code can have a negative impact on the application startup process. This is especially true for Package developers as your code could end up impacting many websites.
```

**Review by sofietoft — COMMENTED** [review]
Looks great @jacksorjacksor ! 💪 

I've made a single suggestion.

**Comment by jacksorjacksor** [comment]
@sofietoft um.... what do we do if these two checks have been pending for a long while....?

**Comment by sofietoft** [comment]
Good question @jacksorjacksor !
Adding a commit to the PR will trigger the checks, so this is what I usually do.

I'm currently not aware of why some of these checks never complete (it happens A LOT!), but I'm pretty sure it's on GitBooks end.

Looks like another commit here turned all the checks green 😁 
I'll get this merged!

Thanks again 🙌

**Comment by jacksorjacksor** [comment]
Thanks @sofietoft !! Always a pleasure to contribute!

**Pull request merged**